charts/fluentd/values.yaml

```diff
@@ -130,7 +130,7 @@ secrets:
   logzioShippingToken: ""
   logzioListener: ""
 
-secretName: logzio-logs-secret
```

Comment:
Instead of deleting entirely `secretName` from the file, I think we should set is to empty string.